cmd/drainer/drainer.toml

```diff
@@ -63,6 +63,11 @@ ignore-schemas = "INFORMATION_SCHEMA,PERFORMANCE_SCHEMA,mysql"
 #
 #replicate-do-db = ["~^b.*","s1"]
 
+# directory of relay logs. Empty string indicates disabling relay log.
```

Comment:
add more explain like only usable when the downstream is tidb/mysql

Reply:
Addressed.